apps/web/package.json

```diff
@@ -67,6 +67,7 @@
     "accept-language-parser": "^1.5.0",
     "async": "^3.2.4",
     "bcryptjs": "^2.4.3",
+    "canvas": "^2.11.0",
```

Comment:
It doesn't seem to be used.
```suggestion
```

Reply:
Not sure why, but without canvas, it was throwing an error. I think it is required with jsdom